types/index.test-d.ts

```diff
@@ -60,7 +60,8 @@ async function main() {
   server.register(replyFrom, fullOptions);
 
   server.get("/v1", (request, reply) => {
-      reply.from();
+      // Verify the return type of reply.from()
+      const newReply: FastifyReply = reply.from();
```

Comment:
I think the code comment is helpful here, since this is the only place that checks the return type of `reply.from(…)`.

Since the code just adds a type verification, and is functionally identical, I figure it's fine to just latch on to an existing handler instead of adding a new one. Let me know if you disagree, though!

Reply:
please use an assertion from tsd here?

Reply:
Ah, that's way better, thanks! I think d9df225 fixed it :smile: